src/com/xilinx/rapidwright/util/RelocateModulesIntoBlackboxes.java

```diff
@@ -61,9 +60,17 @@ public class RelocateModulesIntoBlackboxes {
 	 * @param blackboxes The list of pairs of a black box cell to be filled and its reference INT tile.
 	 *                    The x-coordinate of this INT tile must match that of the cellAnchor.
 	 */
-	public static boolean relocateModuleInsts(Design top, Module mod, String cellAnchor, List<Pair<String, String>> blackboxes) {
+	public static boolean relocateModuleInsts(Design top, Module mod, String cellAnchor, ArrayList<Pair<String, String>> blackboxes) {
```

Comment:
Is there an advantage to making this an `ArrayList`?

Reply:
Changed to List.
Previously, an operation, was outside this method, on the list uses indexing. The op was moved into the method, thus ArrayList was used to make clear of the requirement. Now, the op is changed to not rely on the indexing.